src/bumper.js

```diff
@@ -387,13 +387,14 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
   }
 
   _onTimeUpdate(): void {
-    this._adBreak &&
+    if (this._adBreak && this._bumperState !== BumperState.LOADING) {
```

Comment:
change `_bumperState` to `state`

Reply:
why it metter?

Reply:
it's not, just keep the same naming

Reply:
Isn't it better `state === playing`? seems like `done`/`idle`/`loaded`/`loading` all shouldn't dispatch ad_progress, right?

Reply:
could be LOADED as well

Reply:
what do you mean same naming? `_bumperState` it's the private and `state` it's the public getter